src/firefly/js/ui/ExampleDialog.jsx

```diff
@@ -179,7 +178,7 @@ class FieldGroupTest extends Component {
 
     constructor(props)  {
         super(props);
-        FieldGroupUtils.initFieldGroup('DEMO_FORM',exDialogReducer);
+        dispatchInitFieldGroup('DEMO_FORM',false,null,exDialogReducer);
```

Comment:
This dispatch triggers warnings every time ExampleDialog comes up:
"Warning: setState(...): Cannot update during an existing state transition (such as within `render`). Render methods should be a pure function of props and state."  

Another warning comes up as I click OK in ExampleDialog:

Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the undefined component.
warning @ warning.js:45
getInternalInstanceReadyForUpdate @ ReactUpdateQueue.js:34
ReactUpdateQueue.enqueueSetState @ ReactUpdateQueue.js:181
ReactComponent.setState @ ReactComponent.js:65
(anonymous function) @ ExampleDialog.jsx:193
(anonymous function) @ FieldGroupUtils.js:136
(anonymous function) @ createStore.js:132
dispatch @ createStore.js:131
(anonymous function) @ fftools.js:92535
(anonymous function) @ index.js:88
process @ ReduxFlux.js:209
dispatchHideDialog @ DialogCntlr.js:34
validUpdate @ CompleteButton.jsx:28
(anonymous function) @ CompleteButton.jsx:34
validateResolvedSingle @ FieldGroupUtils.js:17
(anonymous function) @ FieldGroupUtils.js:56

One item of the ticket was understanding what to do about these warnings. What are your current thoughts?